SPECS-EXTENDED/cassandra/Readme

```diff
@@ -0,0 +1,17 @@
+# Downloading files during build time is not allowed
+# apply pre-downloaded cassandra-build-cache which is
+# required for cassandra to build based on ant build.xml
+# Generate cassandra-build-cache when ever update to latest version.
+1. Boot up Mariner Core 2.0
+2. Refer to cassandra spec and install all build required dependenices.
+3. Download cassandra sources and extract under /usr/src
+4. cd /usr/src/apache-cassandra-<version>-src
+5. Run "export JAVA_HOME=/usr/lib/jvm/msopenjdk-11"
+6. Run export ANT_OPTS="-Xmx1024m -XX:MaxPermSize=512m"
+7. Run "sudo ant clean ant -v clean jar -Drelease=true -Duse.jdk11=true"
+8. Once build completed successfully take build cache as below.
+9. cd ~/.m2
+9. tar -czvf cassandra-build-cache-<version>.tar.gz repository
```

Comment:
Suggest to use tar commandline arguments used in other specs to make this reproducible. See - https://github.com/microsoft/CBL-Mariner/blob/9f9217c5f9f55621dec5a188cba780c5c8972a1b/SPECS/azure-iotedge/azure-iotedge.spec#L14

Reply:
updated.